scholia/app/templates/venue.html

```diff
@@ -84,14 +84,24 @@
  LIMIT 200
  `
 
-
- 
+ mostCitedWorksSparql = `
+  # Most cited works published in venue
+  SELECT (COUNT(?citing_work) AS ?count) ?work ?workLabel  WHERE {
+    ?work wdt:P1433 wd:Q6294930.
+    ?citing_work wdt:P2860 ?work.
+    SERVICE wikibase:label { bd:serviceParam wikibase:language "en,da,de,es,fr,jp,nl,no,ru,sv,zh" . } 
```

Comment:
Yes, I saw that... they're also more complicated, but this is the kind of review I was hoping for. I will update that patch accordingly!